marklogic-data-hub-central/ui/src/components/entities/mapping/entity-map-table/entity-map-table.tsx

```diff
@@ -131,7 +131,33 @@ const EntityMapTable: React.FC<Props> = (props) => {
     </ul></div>
   </div>;
 
-  const getColumnsForEntityTable: any = () => {
+  const getColumnsForEntityTable: any = (newTable?: boolean) => {
+    if (newTable) {
+      return entityColumns.map(el => props.checkedEntityColumns[el.key] ? el : "").filter(item => item).map(el => {
+        //console.log("ELEMENT", el);
```

Comment:
all this comment out lines are needed? Othewise clean up

Reply:
oops, sandbox code to be added on DHFPROD-7747, good catch!